docs/samples/Microsoft.ML.Samples/Dynamic/OneHotEncodingTransform.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.OneHotEncodingTransformer;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class OneHotEncodingTransform
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
```

Comment:
>GetInfertData [](start = 101, length = 13)

think we want to deprecate the infert datasets from the samples; as it is a sensitive one. Is it possible to use one of the other dataset snippets?

Reply:
you can use Adult dataset that exists in DatasetUtils

---
In reply to: [261049116](https://github.com/dotnet/machinelearning/pull/2779#discussion_r261049116) [](ancestors = 261049116)